datajs/src/parser/csv.js

```diff
@@ -132,3 +129,44 @@ export class Uint8ArrayToStringsTransformer {
     }
   }
 }
+
+async function getChunkText(file, size = 10) {
```

Comment:
@mariorodeghiero has implemented similar function called [readChunked()](https://github.com/datopian/data.js/blob/master/datajs/src/browser-utils/utils.js#L51). Can we merge those two? Also I would keep this in utils.

Reply:
Okay, will work on that now.